lsp-dart.el

```diff
@@ -120,10 +284,26 @@ PARAMS closing labels notification data sent from WORKSPACE."
                   :initialization-options
                   `((onlyAnalyzeProjectsWithOpenFiles . ,lsp-dart-only-analyze-projects-with-open-files)
                     (suggestFromUnimportedLibraries . ,lsp-dart-suggest-from-unimported-libraries)
-                    (closingLabels . ,lsp-dart-closing-labels))
-                  :notification-handlers (ht ("dart/textDocument/publishClosingLabels" 'lsp-dart--handle-closing-labels))
+                    (closingLabels . ,lsp-dart-closing-labels)
+                    (outline . ,lsp-dart-outline)
+                    (flutterOutline . ,lsp-dart-flutter-outline))
+                  :notification-handlers (ht ("dart/textDocument/publishClosingLabels" 'lsp-dart--handle-closing-labels)
+                                             ("dart/textDocument/publishOutline" 'lsp-dart--handle-outline)
+                                             ("dart/textDocument/publishFlutterOutline" 'lsp-dart--handle-flutter-outline))
                   :server-id 'dart_analysis_server))
 
+;;;###autoload
+(with-eval-after-load 'lsp-treemacs
```

Comment:
all pieces that refer to lsp-treemacs should be wrapped in `(with-eval-after-load` . Without this, the travis will fail.

Reply:
Done, I don't know if I need to wrap the "private" functions given that the public one is already wrapped with `with-eval-after-load`

Reply:
everything that is undefined will cause test failures no matter if it is public or private.